src/open_inwoner/conf/base.py

```diff
@@ -947,7 +947,41 @@
 
 CKEDITOR_CONFIGS = {
     "default": {
+        "height": 200,
+        "width": "80%",
```

Comment:
General: Percentages are (usually) bad and won't account for needed breaks on smaller screens. 
Also: width and height should not be static like this but you could use values like min-height/max-height/max-width in these types of situations.

More specific: we do not need new additions in base? Since we already have this CSS here: `src/open_inwoner/scss/admin/_ck_editor.scss`
For an example of the way we use this wysiwyg editor, you can also peek at any of the cases in http://localhost:8000/admin/openzaak/zaaktypeconfig/ and look at how we treated the 'ZAAKTYPE STATUSTYPE CONFIGURATIES' of each casetype.

And most important: we should only give a limited number of options to the editors.
So this would be the maximum of stuff we should make available really (do not give them the option to add colors or unnecessary textstyles like "typewriter"): 👇

<img width="928" alt="editor-options-casetype" src="https://github.com/maykinmedia/open-inwoner/assets/118177951/37986f18-80a0-46c1-a449-9cf7be82edf8">